crates/symbolicator/src/utils/sentry.rs

```diff
@@ -46,34 +46,26 @@ where
 
     fn call(&mut self, request: Request<Body>) -> Self::Future {
         sentry::configure_scope(|scope| {
-            let (transaction, req) = extract_request(&request);
+            let headers = request.headers();
+            fn get_str(val: &HeaderValue) -> Option<&str> {
+                val.to_str().ok()
+            }
```

Comment:
heh, i would never have thought of writing this as an fn rather than a closure.  but that makes total sense.